### PR TITLE
Server

### PR DESCRIPTION
**Pull request by adispectory**

### CODE REVIEW

**Comment by guyogev** [comment]
Hey. 
Looks like you've been busy :) 
Thinks PR looks very big, I'm sure you most of the files are here by mistake.
Can you please leave only the relevant files?

Thanks

**Comment by adispectory** [comment]
thanks,
I think that now it's better:)

File: config/prod.secret.exs [hunk 1]
```diff
@@ -0,0 +1,20 @@
+use Mix.Config
```
Comment (guyogev) [inline config/prod.secret.exs:1]: prod.secret.exs should never be under git. it contains sensative information.
have a look at https://hexdocs.pm/phoenix/heroku.html#making-our-project-ready-for-heroku.

File: priv/static/js/app.js [hunk 2]
```diff
@@ -0,0 +1,1483 @@
+/******/ (function(modules) { // webpackBootstrap
```
Comment (guyogev) [inline priv/static/js/app.js:1]: priv/static/js/app.js is auto generated on every server build. should not be under git.

File: server/brunch-config.js [hunk 3]
```diff
@@ -0,0 +1,69 @@
+exports.config = {
```
Comment (guyogev) [inline server/brunch-config.js:1]: way do we need brunch? The server is already configed with webpack for assets bundling.

File: server/config/prod.exs [hunk 4]
```diff
@@ -64,5 +66,5 @@ config :logger, level: :info
 config :specto_pusher, SpectoPusher.Repo,
   adapter: Sqlite.Ecto2,
   database: "specto_pusher_prod.sqlite3"
-
-import_config "db/prod.secret.exs"
+  
+import_config "prod.secret.exs"
```
Comment (guyogev) [inline server/config/prod.exs:70]: prefer using ENV vars over prod.secret.exs when deploying to heroku.

File: server/package.json [hunk 5]
```diff
@@ -1,7 +1,10 @@
 {
```
Comment (guyogev) [inline server/package.json:1]: no need to use brunch. we have webpack.

**Review by guyogev — COMMENTED** [review]
Great job. A few high level comments

1. db files should not be under version control. those are private. we need to remove db/specto_pusher_dev.sqlite3 from git.

2. priv/static/ files are compiled on every server build. those should not be under git too.

3. We use webpack to manage assests bundling at the server side. why do we need brunch as well?

**Pull request merged**